GRPCServiceA/pkg/utils/queries.go

```diff
@@ -0,0 +1,5 @@
+package utils
```

Comment:
what if you have  another BD engines? try to name the file being descriptive. "mysql_queries" "mongo_queries" what d you think about?